ansible/sample_production/setup_infrastructure.yml

```diff
@@ -51,12 +51,11 @@
       key_file: "{{ priv_key_file }}"
       state: present # or absent
       name: "{{ item.name }}"
-      display_name: "{{ item.display_name }}"
       availability_zone: "{{ item.availability_zone }}"
-      mtu: "{{ item.mtu }}"
       gateway: "{{ item.gateway }}"
       addresses: "{{ item.address }}"
-      provider_subnets: "{{ item.network_interface_groups }}"
+      network_interface_groups: "{{ item.network_interface_groups }}"
+      endpoint_type: "{{item.endpoint_type}}"
```

Comment:
Don't we still have a display_name field? Should let the users populate that if they want. GUI uses display name quite extensively.